mqtt/mqtt-bridge/src/pump/mod.rs

```diff
@@ -251,7 +243,7 @@ impl<M: 'static> PumpHandle<M> {
     /// Sends a shutdown control message to a pump.
     pub async fn shutdown(mut self) {
         if let Err(e) = self.send(PumpMessage::Shutdown).await {
-            error!(error = %e, "unable to request shutdown for pump.");
+            warn!(error = %e, "unable to request shutdown for pump. probably the pump is about to exit");
```

Comment:
Why can this happen? If there is a succinct way to give more details here then would recommend.

Reply:
we have a bridge handle, which I used to send shutdown message to both pumps when requested. both pumps starting shutdown sequence in parallel. so at some point in time when either pump is finished, another may not but the channel is closed already. in this case we will see an error.
nevertheless we still want to make the pump to stop, when the another one is stopped suddenly (without shutdown signal)